### PR TITLE
Readme Link Fixes

### PR DESCRIPTION
**Pull request by jsotobroad**

### CODE REVIEW

**Comment by ruchim** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -215,8 +215,8 @@ For many examples on how to use WDL see [the WDL site](https://github.com/broadi
   * [String sub(String, String, String)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#string-substring-string-string)
   * [Array\[Int\] range(Int)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#arrayint-rangeint)
   * [Array\[Array\[X\]\] transpose(Array\[Array\[X\]\])](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#arrayarrayx-transposearrayarrayx)
-  * [Pair(X,Y) zip(X,Y)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#pairxy-zipxy)
-  * [Pair(X,Y) cross(X,Y)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#pairxy-crossxy)
+  * [Pair(X,Y) zip(X,Y)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#arraypairxy-ziparrayx-arrayy)
+  * [Pair(X,Y) cross(X,Y)](https://github.com/broadinstitute/wdl/blob/develop/SPEC.md#arraypairxy-crossarrayx-arrayy)
```
Comment (cjllanwarne) [inline README.md]: Looks like the header is wrong too? `[Pair(X,Y) cross(X,Y)]` should be `[List[Pair[X,Y]] cross(List[X], List[Y])]`

**Comment by cjllanwarne** [comment]
Looking at this, it seems strange to me that we've copy/pasted a chunk of the WDL spec's TOC into the Cromwell readme... This is gonna be a nightmare to maintain

File: README.md (hunk 1 above)
Comment (jsotobroad) [inline README.md]: visually it looks fine but ya better to be consistent

**Comment by cjllanwarne** [comment]
Looks good 👍

**Pull request merged**